src/main/java/com/hearthstone/hearthstone_deck_storage/api/HearthstoneClient.java

```diff
@@ -0,0 +1,11 @@
+package com.hearthstone.hearthstone_deck_storage.api;
+
+import com.hearthstone.hearthstone_deck_storage.dto.CardDto;
+
+import java.util.List;
+
+public interface HearthstoneClient {
+
+    List<CardDto> getCardsBySet();
```

Comment:
method names 'by set' but you didn't pass any 'set' to it

Reply:
i use set search in queryparam